fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/metadata/KeyExpressionTest.java

```diff
@@ -766,6 +768,21 @@ void testSplitConcat() {
                 evaluate(splitConcat, numbers));
     }
 
+    @Test
+    void testCanProperlyGetValueExpressionFromKeyWithValue() {
+        /*
+         * Tests for a degenerate case of KeyWithValueExpressions
```

Comment:
Is this really a "degenerate" case of `KeyWithValueExpression`? The test appears to assert that there is one field in the value, which seems like a totally valid usage to me.
If it is, and I'm misunderstanding something about how `KeyWithValueExpression` works, perhaps another reviewer would be better.

Reply:
I think having one value column is definitely a valid use case. I think you could still use "degenerate" here if you really wanted to, but I don't have too many opinions one way or the other